app/views/catalog/_facet_limit.html.erb

```diff
@@ -1,4 +1,11 @@
 <% # Overridden from Blacklight to use   %>
+<% if article_search? && facet_field.key == 'eds_publication_type_facet'%>
+  <% if facet_field_in_params?('eds_publication_type_facet') %>
```

Comment:
@mejackreed After going back to my PR, I realized my 0 results behavior wasn't working properly! I added this line to make sure that the warning only shows up if 1 or more source types are selected.